### PR TITLE
Feature #140 - Add serializer for order post method

### LINKED ISSUE #140: SP: 3 Add serializer for Order POST method
Part of user story #37

### PR DESCRIPTION
**Pull request by Misha86**

### CODE REVIEW

File: beauty/api/serializers/business_serializers.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from rest_framework import serializers
+from api.models import Business, CustomUser
+
+
+class BusinessListCreateSerializer(serializers.ModelSerializer):
```
Comment (pazuzu-ua) [inline beauty/api/serializers/business_serializers.py:5]: Please, add docstring

File: beauty/api/serializers/business_serializers.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    class Meta:
```
Comment (pazuzu-ua) [inline beauty/api/serializers/business_serializers.py:6]: Please, add docstring

File: beauty/api/serializers/business_serializers.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        model = Business
+        fields = ('name', 'type', 'owner', 'description')
+
+    def to_representation(self, instance):
```
Comment (pazuzu-ua) [inline beauty/api/serializers/business_serializers.py:10]: Please, add docstring

File: beauty/api/views.py [hunk 4]
```diff
@@ -89,4 +90,31 @@ def get_object(self):
                                 Q(specialist=self.kwargs['user']),
                                 id=self.kwargs['id'])
         self.check_object_permissions(self.request, obj)
-        return obj
\ No newline at end of file
+        return obj
+
+
+class BusinessListCreateView(ListCreateAPIView):
```
Comment (pazuzu-ua) [inline beauty/api/views.py]: Please, add docstring

File: beauty/api/views.py [hunk 5]
```diff
@@ -89,4 +90,31 @@ def get_object(self):
...
+    queryset = Business.objects.all()
+    serializer_class = BusinessListCreateSerializer
+
+    def get_permissions(self):
```
Comment (pazuzu-ua) [inline beauty/api/views.py]: Please, add docstring

File: beauty/api/serializers/business_serializers.py (hunk 1 above)
Comment (Misha86) [reply beauty/api/serializers/business_serializers.py:5]: Business is not my task. I removed it from my branch

File: beauty/api/serializers/business_serializers.py (hunk 2 above)
Comment (Misha86) [reply beauty/api/serializers/business_serializers.py:6]: Business is not my task. I removed it from my branch

File: beauty/api/serializers/business_serializers.py (hunk 3 above)
Comment (Misha86) [reply beauty/api/serializers/business_serializers.py:10]: Business is not my task. I removed it from my branch

File: beauty/api/views.py (hunk 4 above)
Comment (Misha86) [reply beauty/api/views.py]: Business is not my task. I removed it from my branch

File: beauty/api/views.py (hunk 5 above)
Comment (Misha86) [reply beauty/api/views.py]: Business is not my task. I removed it from my branch

**Pull request closed without merging**